### PR TITLE
Development

### PR DESCRIPTION
**Pull request by KateMcRo**
First live test of dev branch on Heroku.

### CODE REVIEW

File: client/public/index.html
Comment (pvmend) [inline client/public/index.html:1]: Doctype is ok even though it's lowercased?

File: client/src/pages/Login.jsx
Comment (pvmend) [inline client/src/pages/Login.jsx:1]: I still need help with the login button redirecting and signing in user

File: client/public/index.html
Comment (KateMcRo) [reply client/public/index.html:1]: The case doesn't effect functionality. We can update it later to reflect best practices/readability.

File: client/src/pages/Login.jsx
Comment (KateMcRo) [reply client/src/pages/Login.jsx:1]: I made a note in the approval that this was a WIP. I just want to test the env variables in Heroku so we don't have any last minute surprises to deal with.

File: client/src/pages/Search.jsx
Comment (pvmend) [inline client/src/pages/Search.jsx:1]: Should we have the api call in a separate file in utils?

Comment (KateMcRo) [reply client/src/pages/Search.jsx:1]: I can definitely add that to the list.

File: client/src/pages/Signup.jsx
Comment (pvmend) [inline client/src/pages/Signup.jsx:1]: may need someone to look over the signup redirect and are we going to use usernames ? the login only requires a email and password so we can probably get rid of the first name and last name text field and not add username.

Comment (Jrossi425) [reply client/src/pages/Signup.jsx:1]: yea email and password only I think

**Review by pvmend — APPROVED** [review]
left some comments on the doctype being lowercased and about our API call

**Review by Jrossi425 — APPROVED** [review]

File: client/src/pages/Signup.jsx
Comment (KateMcRo) [reply client/src/pages/Signup.jsx:1]: The user model currently has username and its included in the "my garden" view as part of a greeting.

**Review by ManinderManan — APPROVED** [review]

**Pull request merged**